.env

```diff
@@ -0,0 +1,3 @@
+MONGODB=mongodb://localhost/asyncvoter
```

Comment:
I changed the behaviour to edit locally because the file needs to exist in order for travois ci to be able to execute the tests